FsSonarRunner/FsSonarRunner/App.config

```diff
@@ -1,18 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
we might needed these redirects. likely the option is disable now in project file. auto genererate bind redirects should be set in project.

Reply:
At least I was able to run `FsSonarRunner` on `FsSonarRunner.sln` and fixed the issues found in commit 7048ca2ba01a940bc171f391de1051b80b310f56 straightaway.